tesla-ble.example.yml

```
@@ -1,25 +1,117 @@
 
 packages:
-  base: !include packages/base.yml
-  client: !include packages/client.yml
-  common: !include packages/common.yml
-  # listener: !include packages/listener.yml # Uncomment this to scan find your VIN BLE MAC address
-  # for ESPHome Dashboard, use this instead of external_components
+  client:
+    url: https://github.com/mrd0n/esphome-tesla-ble/
+    ref: main
+    files: packages/client.yml
+    refresh: 0s # This seems to not have affect;
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)